docs/index.md

```diff
@@ -39,7 +39,8 @@ Install using `pip`:
 $ pip install uvicorn
 ```
 
-This will install uvicorn with minimal (pure Python) dependencies.
+This will install uvicorn with minimal (pure Python) dependencies 
+(note: this does not enable websockets support, see below for "activating" it).
```

Comment:
I find this note a bit confusing... FWIW, I think the existing prose here is sufficient, and the new warning makes it even easier for users to see that they should use the standard install for WebSocket support.

Reply:
ok I removed it